pocs/observatory.py

```diff
@@ -203,6 +204,15 @@ def remove_camera(self, cam_name):
         self.logger.debug('Removing {}'.format(cam_name))
         del self.cameras[cam_name]
 
+    def add_scheduler(self, scheduler):
+        assert isinstance(scheduler, BaseScheduler)
```

Comment:
This will throw an `AssertionError` at the user, which could cause weird things to happen. Let's just check `if isinstance` and give a warning if not.